system/doc/general_info/DEPRECATIONS

```diff
@@ -74,6 +74,8 @@ snmpm:async_get_bulk/6 since=23 remove=25
 snmpm:async_get_bulk/7 since=23 remove=25
 snmpm:async_get_bulk/8 since=23 remove=25
 
+erlang:phash/2 since=23
```

Comment:
Minor nit: this should be under "Added in OTP 24" with `since=24` if you're deprecating it in master.

Reply:
Thanks for finding this! Fixed: 4ba65a0